bin/codemods/README.md

```diff
@@ -67,6 +67,24 @@ If you're developing your own transformations, it may be useful to know you can
 ./node_modules/.bin/jscodeshift -t transformation.js [target files]
 ```
 
+## How to debug codemods
+
+If you are a codemod author, you may want to debug your codemod using the Chrome debugger. Then
+run the codemod script with a `--debugger` parameter:
+```bash
+./bin/codemods/run.js --debugger my-transform client/target.js
```

Comment:
I think for the docs we can omit this and just include a canonical `npm run codemod …`. The savvy can figure out the direct invocation if they're interested.